definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -4737,5 +4737,33 @@
     "CaseEventID": "solicitorSOTDN",
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solConfirmPersonalService",
+    "UserRole": "[PETSOLICITOR]",
```

Comment:
Hi Mounika - we have withdrawn the [PETSOLICITOR] role until after we can fully test this area, given we're not pursuing the current approach of linking. As it is not in the next release (probably) then we should hold back merging this stuff...

Reply:
Hi @alexwakeman  will make a note of it.  Do we need to use normal caseworker solicitor role in this case? Will check with @gavinclark  before merging this.

Reply:
Would use caseworker-divorce-solicitor as we would have done previously

Reply:
> Would use caseworker-divorce-solicitor as we would have done previously

Moved the permissions to caseworker-divorce-solicitor role